backend/app/api/routes/doc_transformation_job.py

```
@@ -0,0 +1,45 @@
+
+logger = logging.getLogger(__name__)
+router = APIRouter(prefix="/documents/transformations", tags=["doc_transformation_job"])
+
+@router.get(
+    "/{job_id}",
+    description="Get the status and details of a document transformation job.",
+    response_model=APIResponse,
+)
+def get_transformation_job(
+    session: SessionDep,
+    current_user: CurrentUser,
```

Ruff (0.12.2)

Unused function argument: `current_user`

(ARG001)